Google.Api.Generator.Tests/ProtoTests/Lro/Testing.Lro.Snippets/Testing.Lro.Snippets.csproj

```diff
@@ -0,0 +1,14 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netcoreapp2.2;net452</TargetFrameworks>
+    <TargetFrameworks Condition=" '$(OS)' != 'Windows_NT' ">netcoreapp2.2</TargetFrameworks>
```

Comment:
Any chance we could remove the spaces at the start and end of the string? I'm fine with the ones between the values and `!=`, but the start and end ones always bug me :)

Reply:
Done :)